lib/jekyll-relative-links/generator.rb

```diff
@@ -23,15 +26,19 @@ def initialize(site)
     def generate(site)
       @site    = site
       @context = context
+      return if disabled?
 
-      site.pages.each do |page|
+      pages = site.pages
+      pages = site.pages + site.documents if collections?
```

Comment:
Should `documents` really be `docs_to_write`?

Reply:
yes it should! I wasn't aware of this property until just now, but it does exactly what we want.